src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1162,6 +1156,21 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         _clipboardCopyHandlers(copiedData);
     }
 
+    // Method Description:
+    // - Initiate a paste operation.
+    // Arguments:
+    // - <none>

```

Comment:
nit: please remove line 1161/1162 since we don't really need them. :)

Reply:
Fixed in commit